floris/tools/optimization/scipy/yaw_wind_rose.py

```diff
@@ -536,33 +540,34 @@ def calc_baseline_power(self):
         result_dict = dict()
 
         for i in range(len(self.wd)):
-            if self.ti is None:
-                print(
-                    "Computing wind speed, wind direction pair "
-                    + str(i)
-                    + " out of "
-                    + str(len(self.wd))
-                    + ": wind speed = "
-                    + str(self.ws[i])
-                    + " m/s, wind direction = "
-                    + str(self.wd[i])
-                    + " deg."
-                )
-            else:
-                print(
-                    "Computing wind speed, wind direction, turbulence "
-                    + "intensity set "
-                    + str(i)
-                    + " out of "
-                    + str(len(self.wd))
-                    + ": wind speed = "
-                    + str(self.ws[i])
-                    + " m/s, wind direction = "
-                    + str(self.wd[i])
-                    + " deg, turbulence intensity = "
-                    + str(self.ti[i])
-                    + "."
-                )
+            if self.verbose:
+                if self.ti is None:
+                    print(
+                        "Computing wind speed, wind direction pair "
+                        + str(i)
+                        + " out of "
+                        + str(len(self.wd))
+                        + ": wind speed = "
+                        + str(self.ws[i])
+                        + " m/s, wind direction = "
+                        + str(self.wd[i])
+                        + " deg."
+                    )
```

Comment:
Something to think about for the future (but this seemed like a nice way to bring it up) would be to convert a lot of this string concatenation to f-strings (new to Python 3.4). Basically, it allows string formatting to look more like the text it's supposed to reading out as, and this would become:

```python
f"Computing wind speed, wind direction pair {i} out of {len(self.wd)}: wind speed = {self.ws[i]} m/s, wind direction = {self.wd[i])} deg."
# this can also be broken down into two lines to fit within the line length as well
text_to_print = (
    f"Computing wind speed, wind direction pair {i} out of {len(self.wd)}: ",
    f"wind speed = {self.ws[i]} m/s, wind direction = {self.wd[i])} deg"
)
```